backend/tests/integration/test_history_integration.py

```
@@ -661,22 +661,79 @@ def test_api_session_detail_integration(self, client, mock_history_service_for_a
                 "status": "completed",
                 "started_at_us": current_time_us - 300000000,  # Started 5 minutes ago
                 "completed_at_us": current_time_us,  # Completed now
-                "error_message": None
+                "error_message": None,
+                "chain_id": "integration-chain-123"  # Add chain_id so endpoint processes chain execution
             },
             "chronological_timeline": [
                 {
                     "interaction_id": "int-1",
-                    "type": "llm_interaction",
+                    "type": "llm",  # Changed from llm_interaction to llm to match controller logic
```

Ruff (0.12.2)

Line too long (99 > 88)

(E501)